src/app/types/SøknadTempStorageData.ts

```diff
@@ -1,7 +1,7 @@
 import { StepID } from '../søknad/søknadStepsConfig';
 import { SøknadFormData } from './SøknadFormData';
 
-export const MELLOMLAGRING_VERSION = '12.0.0';
+export const MELLOMLAGRING_VERSION = '12.0.1';
```

Comment:
Trengs det egentlig å endre versjonsnummer for denne endringen? Endringen vil vel berøre så få, at konsekvensen er større ved å endre versjonsnummer enn å ikke gjøre det?